internal/runners/show/show.go

```diff
@@ -232,36 +244,37 @@ func languagesData(owner, project string) ([]string, error) {
 
 func visibilityData(owner, project string, remoteProject *mono_models.Project) string {
 	if remoteProject.Private {
-		return "Private"
+		return locale.T("show_visibility_private")
 	}
-	return "Public"
+	return locale.T("show_visibility_public")
```

Comment:
If you're not gonna use `locale.Tl` then make the key less explicit, just call it "private" and "public".